python/ray/tests/test_debug_tools.py

```diff
@@ -47,3 +47,12 @@ def f():
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE)
         assert pgrep_command.communicate()[0]
+
+
+if __name__ == "__main__":
+    import pytest
+    import sys
+    # Make subprocess happy in bazel.
+    os.environ["LC_ALL"] = "en_US.UTF-8"
+    os.environ["LANG"] = "en_US.UTF-8"
```

Comment:
Setting these are necessary for some tests since bazel erases the env variables from outside.